src/source/Metrics/Metrics.c

```diff
@@ -72,13 +72,25 @@ STATUS getTransportStats(PRtcPeerConnection pRtcPeerConnection, PRtcTransportSta
     return retStatus;
 }
 
-STATUS getRtpRemoteInboundStats(PRtcPeerConnection pRtcPeerConnection, PRtcRemoteInboundRtpStreamStats pRtcRemoteInboundRtpStreamStats)
+STATUS getRtpRemoteInboundStats(PRtcPeerConnection pRtcPeerConnection, PRtcRtpTransceiver pTransceiver,
+                                PRtcRemoteInboundRtpStreamStats pRtcRemoteInboundRtpStreamStats)
 {
     STATUS retStatus = STATUS_SUCCESS;
     PKvsPeerConnection pKvsPeerConnection = (PKvsPeerConnection) pRtcPeerConnection;
     UNUSED_PARAM(pKvsPeerConnection);
     CHK(pRtcPeerConnection != NULL || pRtcRemoteInboundRtpStreamStats != NULL, STATUS_NULL_ARG);
-    CHK(FALSE, STATUS_NOT_IMPLEMENTED);
+    PKvsRtpTransceiver pKvsRtpTransceiver = (PKvsRtpTransceiver) pTransceiver;
+    if (pKvsRtpTransceiver == NULL) {
+        PDoubleListNode node = NULL;
+        CHK_STATUS(doubleListGetHeadNode(pKvsPeerConnection->pTransceievers, &node));
+        CHK_STATUS(doubleListGetNodeData(node, (PUINT64) &pKvsRtpTransceiver));
+        CHK(pKvsRtpTransceiver != NULL, STATUS_NOT_FOUND);
```

Comment:
So in case the user doesn't specify the transceiver, we use the first one? Do you mind adding a comment?